ampersand-select-view.js

```diff
@@ -65,7 +65,7 @@ function SelectView (opts) {
 
     this.render();
 
-    this.setValue(opts.value);
+    opts.value && this.setValue(opts.value);
```

Comment:
This is slightly outdated ticket and there has been some work going on around it anyway. I'll close it as I need to write proper one with tests anyway. Thanks.